dispatcher/backend/src/common/entities.py

```diff
@@ -15,9 +18,62 @@ def all(cls):
 
 
 class ScheduleCategory:
-    wikipedia = 'wikipedia'
+    gutenberg = 'gutenberg'
+    other = 'other'
     phet = 'phet'
+    psiram = 'psiram'
+    stack_exchange = 'stack_exchange'
+    ted = 'ted'
+    vikidia = 'vikidia'
+    wikibooks = 'wikibooks'
+    wikinews = 'wikinews'
+    wikipedia = 'wikipedia'
+    wikiquote = 'wikiquote'
+    wikisource = 'wikisource'
+    wikispecies = 'wikispecies'
+    wikiversity = 'wikiversity'
+    wikivoyage = 'wikivoyage'
+    wiktionary = 'wiktionary'
+
+    @classmethod
+    def all(cls):
+        return [cls.gutenberg, cls.other, cls.phet, cls.psiram, cls.stack_exchange,
+                cls.ted, cls.vikidia, cls.wikibooks, cls.wikinews, cls.wikipedia,
+                cls.wikiquote, cls.wikisource, cls.wikispecies, cls.wikiversity,
+                cls.wikivoyage, cls.wiktionary]
+
+    @classmethod
+    def get_warehouse_path(cls, category):
+        return '/{}'.format(category)
+
+    @classmethod
+    def all_warehouse_paths(cls):
+        return [cls.get_warehouse_path(category) for category in cls.all()]
+
+
+class ScheduleQueue:
+    small = 'small'
+    medium = 'medium'
+    large = 'large'
+    debug = 'debug'
 
     @classmethod
     def all(cls):
-        return [cls.wikipedia, cls.phet]
+        return [cls.small, cls.medium, cls.large, cls.debug]
+
+
+class ScheduleLanguage:
```

Comment:
This class is not used. I personally think it is generally a good idea to add then until you need them.